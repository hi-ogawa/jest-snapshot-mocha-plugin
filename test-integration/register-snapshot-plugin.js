const process = require("process");
const { JestSnapshotMochaPlugin } = require("../build/tsc/src");

const plugin = new JestSnapshotMochaPlugin({
  updateSnapshot: !!process.env.UPDATE_SNAPSHOT,
});

global.matchSnapshot = plugin.matchSnapshot;
exports.mochaHooks = plugin.mochaHooks;
