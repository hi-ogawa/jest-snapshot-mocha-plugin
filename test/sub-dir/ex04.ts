import { describe, it } from "mocha";
import { matchSnapshot } from "../register-snapshot-plugin";

describe("ex04", () => {
  it("sub-dir", () => {
    matchSnapshot(18181);
  });
});
