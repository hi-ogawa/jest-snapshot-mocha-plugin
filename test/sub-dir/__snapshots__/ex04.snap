// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ex04 sub-dir 1`] = `18181`;
